src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -129,15 +210,16 @@ private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appCo
                         filterIndex++;
 
                         //
-                        // Handle AlwaysOn filters
-                        if (string.Equals(featureFilterConfiguration.Name, "AlwaysOn", StringComparison.OrdinalIgnoreCase))
+                        // Handle AlwaysOn and On filters
+                        if (string.Equals(featureFilterConfiguration.Name, "AlwaysOn", StringComparison.OrdinalIgnoreCase) ||
+                                string.Equals(featureFilterConfiguration.Name, "On", StringComparison.OrdinalIgnoreCase))
```

Comment:
Should we mention that the filter name "On" is also occupied and it is a built-in filter in the document.